src/components/BookList.js

```diff
@@ -17,22 +20,41 @@ const BookList = ({ books }) => {
   };
 
   return (
-    <ul>
+    <div className="book-list">
       {books.map((book) => (
-        <li key={book.itemId || uuidv4()}>
-          <h3>{book.title}</h3>
-          <p>
-            Author:
-            {book.author}
-          </p>
-          <p>
-            Category:
-            {book.category}
-          </p>
-          <button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove Book</button>
-        </li>
+        <div key={book.itemId || uuidv4()} className="book-details">
+          <div className="book-info">
+            <p className="School-of Text-Style-9">
+              {book.category}
+            </p>
+            <h3 className="Title Text-Style-5">{book.title}</h3>
+            <div className="author-buttons">
+              <p className="Suzanne-Collins Text-Style-8">
+                {book.author}
+              </p>
+              <ButtonGroup variant="text" aria-label="text button group">
+                <Button type="submit">Comments</Button>
+                <Button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove</Button>
+                <Button type="submit">Edit</Button>
+              </ButtonGroup>
+            </div>
```

Comment:
- [ ] Regarding the booklist I think the background is white in the book list and under the list is #FAFAFA so I think it would be great if you make the background for each list white and the space under the booklist #FAFAFA this color so please make it to have it the same as the given design ⭐⭐🔥💯


- I think you made the opposite ⬇
- Your
![image](https://github.com/Eccb7/book-store/assets/58553711/8dbdaffc-ea89-40ae-9236-18a34d63e508)


- Given
![image](https://github.com/Eccb7/book-store/assets/58553711/4852f8de-8333-43bd-a875-8a27b84ad130)